src/app/pages/about/index.tsx

```diff
@@ -8,7 +8,7 @@ function About() {
             <Header as="h1">Açık Kaynak İnisiyatifi Hakkında</Header>
 
             <p>
-                Bu organizasyon Türkiye'deki ve Türkçe konuşan geliştiricilerin oluşturduğu
+                Bu organizasyon Türkçe konuşan ve Türkiye'deki geliştiricilerin oluşturduğu
```

Comment:
Bunun gramer olarak bir gerekçesi var mı ?

Reply:
Benim bildiğim kadarıyla yok. Fakat genel olarak okunması daha rahat(kendimce) olduğu için böyle bir değişikliği yolladım.

Reply:
Bana göre de sanki okunurken daha iyi olmuş bu şekilde.